### ISSUE TITLE
confusion about train_set and dev_set

### ISSUE THREAD

**Issue by PipiZong** [castorini/honk#95]
How to concatenate train_set and dev_set?

**Comment by ljj7975** [comment]
try setting `--dev_pct 0`
can you tell us why you need to combine the two?

**Comment by PipiZong** [comment]
Thank you for your reply.
I know setting --dev_pct 0 can work. But I want to split the data into three set, and then combine the training set and validation set (dev_set) as a new training set and train another model with the whole new training data. Do you have any better idea that can recommend to me? Looking forward to your reply.

**Comment by daemon** [comment]
That explains _what_ you want to do, not _why_ you want to do it.

A better way to combine the two splits would be to extend `torch.utils.Dataset` and rewrite the dataset splitting logic for the Speech Commands dataset. You can take a look at the (admittedly messy) `utils/model.py` for a general idea.
